packages/example-app-base/src/components/ComponentPage/ComponentPage.tsx

```diff
@@ -107,44 +106,11 @@ export class ComponentPageBase extends React.PureComponent<IComponentPageProps>
     );
   }
 
-  private _getNativePropsInfo(): JSX.Element | undefined {
-    const { allowNativeProps, allowNativePropsForComponentName, nativePropsElement = 'div' } = this.props;
-    if (allowNativeProps) {
-      const nativePropsElems = Array.isArray(nativePropsElement) ? nativePropsElement : [nativePropsElement];
-
-      const elementsArr: (JSX.Element | string)[] = [];
-      for (const elem of nativePropsElems) {
-        elementsArr.push(<code key={elem}>{`<${elem}>`}</code>);
-        elementsArr.push(' and ');
-      }
-      elementsArr.pop(); // remove last ' and '
-      elementsArr.push(` tag${nativePropsElems.length > 1 ? 's' : ''}`);
-
-      let componentNameJsx: JSX.Element | undefined;
-      if (allowNativePropsForComponentName) {
-        componentNameJsx = <code>{allowNativePropsForComponentName}</code>;
-      }
-
-      return (
-        <MessageBar>
```

Comment:
Are we not needing the message bar with native props support message?

Reply:
I figured we don't really need this anymore because we have inheritance documented in the interface/enum descriptions now. Like this:

<img width="888" alt="Screen Shot 2019-04-16 at 1 49 46 PM" src="https://user-images.githubusercontent.com/14134000/56243094-9c72de00-604e-11e9-944c-8eb7560667b5.png">

Reply:
Ohhh that makes sense 👍